lib/firebase/fetch_survey.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+
+// Data model for a survey question
+class SurveyQuestion {
+  final String id;
+  final String text;
+
+  SurveyQuestion({required this.id, required this.text});
+
+  factory SurveyQuestion.fromSnapshot(DocumentSnapshot snapshot) {
+    return SurveyQuestion(
+      id: snapshot.id,
+      text: snapshot['text'],
+    );
+  }
+}
+
+// Firebase service to interact with Firestore
+class FirebaseService {
+  final FirebaseFirestore _firestore = FirebaseFirestore.instance;
```

Comment:
In the branch I'm working on, the Firestore instance is saved as a [global variable](https://github.com/theheartcenter-one/thc/blob/577f4e3328e2a7540acee7814d8def5c220ddceb/lib/firebase/firebase.dart#L23), so eventually we'll probably use that variable here too.